### DISCUSSION TITLE
Plex server access to Nexcloud on local

### DISCUSSION THREAD

**Discussion by vawaver** [nextcloud/all-in-one#2766] [Questions] [answered]
I have the Plex Media Server Docker installed, along with Nextcloud AIO Docker, on the same Ubuntu Server 20.04.6. I want to know if it's possible to configure the Plex server to access the directory containing media content in Nextcloud locally.

Previously, I had Nextcloud installed as a snap on the same Ubuntu server, and I was able to set up a direct path to the directory in question. 
How can I achieve the same setup now?

**Comment by szaimen** [comment, marked as answer]
Hi, of course this is possible. By default Nextcloud files are stored in the nextcloud_aio_nextcloud_data volume if you did not specify a different directory with NEXTCLOUD_DATADIR. Ownership of these files is 33:0. For further help on this please refer to https://help.nextcloud.com
